### PR TITLE
Reuse existing subscriptions

### PR DESCRIPTION
**Pull request by AndersGM**
https://app.asana.com/0/1153435032856801/1199376127375581

### CODE REVIEW

File: ruby-gem/lib/api_maker/javascript/cable-connection-pool.js [hunk 1]
```diff
@@ -10,168 +11,192 @@ export default class ApiMakerCableConnectionPool {
...
+        this.upcomingSubscriptionData[modelName]["events"][eventName].push(modelId)
 
-    if (!("model_class_events" in this.upcomingSubscriptionData[modelName]))
-      this.upcomingSubscriptionData[modelName]["model_class_events"] = []
+      if (!this.upcomingSubscriptions[modelName])
+        this.upcomingSubscriptions[modelName] = {}
 
-    if (!this.upcomingSubscriptionData[modelName]["model_class_events"].includes(eventName))
-      this.upcomingSubscriptionData[modelName]["model_class_events"].push(eventName)
+      if (!this.upcomingSubscriptions[modelName]["events"])
```
Comment (kaspernj) [inline ruby-gem/lib/api_maker/javascript/cable-connection-pool.js:111]: Why was this changed? It doesn't seem like it was necessary?

**Review by kaspernj — COMMENTED** [review]
I am not sure how this ended up in so many lines of code?

**Review by kaspernj — CHANGES_REQUESTED** [review]
We need a ton of specs for this. This is very hard to review because of all the irrelevant changes.

File: ruby-gem/lib/api_maker/javascript/cable-connection-pool.js (hunk 1 above)
Comment (AndersGM) [reply ruby-gem/lib/api_maker/javascript/cable-connection-pool.js:111]: There is added a condition https://github.com/kaspernj/api_maker/pull/432/files#diff-960a4d8e1fbcdce8c8ca074c9ac52f5b42870df68d5c66988cfb656903e7dcbfR93 that checks whether the subscription can be added to an existing connection. So this code had to be indented - if that is the question?

**Comment by kaspernj** [comment]
Let stop working on this for now :-)

**Pull request closed without merging**